### PR TITLE
Feature/forgot password

### PR DESCRIPTION
**Pull request by skellertor**

### CODE REVIEW

File: src/components/ForgotPassword/ForgotPassword.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  request: START,
+  email: '',
+  error: false,
+};
+
+class ForgotPassword extends React.Component {
+  state = { ...initialState };
+
+  validateEmail(email) {
+    const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (Cody123Anderson) [inline src/components/ForgotPassword/ForgotPassword.js]: We should put this in a util to be used in the other forms as well.

File: src/components/ForgotPassword/ForgotPassword.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  };
+
+  handleEmail = (e) => this.setState({
+    email: e.target.value,
+    error: false
+  });
+
+  renderContent() {
+    const { error, errorMessage, request } = this.state;
+    switch (request) {
```
Comment (Cody123Anderson) [inline src/components/ForgotPassword/ForgotPassword.js]: I like this!

File: src/components/root/routes/Routes.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ import LandingPage from '../../LandingPage';
 import Register from '../../auth/Register';
 import SignIn from '../../auth/SignIn';
 import SwagRoutes from './SwagRoutes';
+import ForgotPassword from '../../ForgotPassword/ForgotPassword';
```
Comment (Cody123Anderson) [inline src/components/root/routes/Routes.js]: We should put the forgot password and reset password components in the `auth` folder

File: src/constants/routes.js [hunk 4]
```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  resetPassword: '/reset',
```
Comment (Cody123Anderson) [inline src/constants/routes.js]: `reset-password` would be nice so it matches forgotPassword and is a little more clear

File: src/components/root/routes/Routes.js (hunk 3 above)
Comment (skellertor) [reply src/components/root/routes/Routes.js]: 👍

File: src/constants/routes.js (hunk 4 above)
Comment (skellertor) [reply src/constants/routes.js]: 👍

File: src/components/ForgotPassword/ForgotPassword.js (hunk 2 above)
Comment (skellertor) [reply src/components/ForgotPassword/ForgotPassword.js]: 👍

File: src/components/ForgotPassword/ForgotPassword.js (hunk 1 above)
Comment (skellertor) [reply src/components/ForgotPassword/ForgotPassword.js]: moved 👍

**Pull request merged**